runtime/lua/vim/health.lua

```diff
@@ -307,32 +308,37 @@ local function shellify(cmd)
   return table.concat(escaped, ' ')
 end
 
+--- @private
```

Comment:
The reason I didn't use `@private` is because the description made it sound like it's for class members only https://luals.github.io/wiki/annotations/#private. Maybe it's just a bad description though?

Reply:
I agree that [`@private`](https://luals.github.io/wiki/annotations/#private) not a valid LuaCATS annotation. Neither `@package` is (A packaged function cannot be accessed from another file). But such functions can be simply defined as `local`. I think, throughout the neovim codebase, `@private` `@package` are being abused and we were using these annotations to simply marking functions as *not exposed for a public use*. I don't think there is a strict, and valid annotations in LuaCATS for this purpose...

Reply:
Eh, I think it should be fine. Let's keep it.

Reply:
> I think, throughout the neovim codebase, @private @package are being abused and we were using these annotations to simply marking functions as not exposed for a public use.

Not abused. We were using them for both LuaLS and our docgen.

We use `@package` on class methods which are accessed within the same file. We use `@private` on class methods and module fields, however LuaLS only provides diagnostics for use on class methods.